src/modules/EventuateClient.js

```diff
@@ -98,16 +99,20 @@ export default class EventuateClient {
         return result.failure(new Error('Incorrect input parameters for create()'));
       }
 
-      const events = this.prepareEvents(_events);
-      const jsonData = {
-        entityTypeName,
-        events
-      };
+      const { encryptionKeyId, ...rest } = options;
+      options = rest;
 
+      let events = this.prepareEvents(_events);
+
+      if (encryptionKeyId && this.encryption) {
+        // Encrypt event data;
+        events = this.encryptEvens(encryptionKeyId, events);
```

Comment:
`encryptEvents()` is mispelt

Move the `if (encryptionKeyId && this.encryption)` into `encryptEvents()` so the code is simply `events = `encryptEvents(events)`